packages/algorithm/package.json

```diff
@@ -26,9 +26,9 @@
     "api": "api-extractor run --local --verbose",
     "build": "npm run build:src && rollup -c",
     "build:src": "tsc --build",
-    "build:test": "tsc --build tests && cd tests && webpack",
+    "build:test": "npm run clean:test && tsc --build tests && cd tests && rollup -c",
```

Comment:
Should clean be part of the build step? If you don't run clean and delete the `tests/tsconfig.tsbuildinfo` file then the build step will do nothing.